pkg/pods/manifest.go

```diff
@@ -93,6 +90,10 @@ func ManifestFromBytes(bytes []byte) (*Manifest, error) {
 		bytes = signed.Plaintext
 	}
 
+	// Preserve the raw manifest so that manifest.Bytes() returns bytes in
+	// the same order that they were passed to this function
+	manifest.raw = make([]byte, len(bytes))
+	copy(manifest.raw, bytes)
```

Comment:
This means the bytes will always be stripped of the signature. (notice how the `manifest.raw` is assigned the signed bytes inside the if statement before the plaintext is extracted)